packages/http-client-js/src/components/operation-handlers/paging/page-response.tsx

```diff
@@ -27,6 +27,8 @@ export function PageResponseDeclaration(props: PageResponseProps) {
     "firstLink",
     "lastLink",
   ];
+  // TODO: consider the nested response
+  // https://github.com/microsoft/typespec/issues/7787
```

Comment:
Bypass nested paging compile issue and created an issue https://github.com/microsoft/typespec/issues/7787 to track the effort.